region/brinstar/green.json

```diff
@@ -2569,17 +2576,22 @@
               "strats": [
                 {
                   "name": "Etecoon E-Tank Beetom Clip",
-                  "notable": false,
+                  "notable": true,
                   "requires": [
-                    "canBeetomIceClip",
+                    "h_canNonTrivialCeilingClip",
+                    "canTrickyUseFrozenEnemies",
                     "Morph",
                     {"enemyDamage": {
                       "enemy": "Beetom",
                       "type": "contact",
                       "hits": 2
                     }}
                   ],
-                  "note": "Two Beetom hits are assumed, but with failed attempts it could take more, possibly requiring leaving the Beetom and going back to the right to farm.",
+                  "note": [
+                    "Jump and freeze the Beetom at a precise location in order to jump through the crumble blocks.",
+                    "The pixel window is larger and higher with Morph and an X-Ray Stand Up.",
```

Comment:
I just noticed that you need another way to gain height to get on top of the beetom once frozen.  With the precise clip, you can rub the frozen enemy to gain height, and conveniently put you in position to clip through the crumbles.  For Morph+Xray you could use a second beetom to stand on but that would also come with more beetom hits, and maybe you farm that health back anyway.

My old setup walljumped up, but this walljumpless setup is working better 😺